src/server.ts

```diff
@@ -23,32 +26,49 @@ const swaggerSpec = swaggerJsDoc(options);
 app.use('/api/docs', swaggerUI.serve, swaggerUI.setup(swaggerSpec));
 
 app.use('/api', routes);
+app.use('/docs', swaggerUI.serve, swaggerUI.setup(swaggerSpec));
 
+// Logging middleware
 app.use(
   expressWinston.logger({
     winstonInstance: logger,
     statusLevels: true,
   })
 );
 
-app.get('/', (req: Request, res: Response): void => {
-  logger.error('Testing logger');
-  res.send('Welcome at Mavericks E-commerce Website Apis');
+// Profile routes
+app.use('/profiles', profileRoutes);
+
+// Error handling middleware for profile routes
+app.use((err: any, req: Request, res: Response, next: NextFunction) => {
+  console.error(err.stack);
+  res.status(500).send('Internal Server Error');
 });
```

Comment:
Why did you add this? You should remove everything you added in the server.ts, I don't think your task involves changing anything from this file